.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build-and-test:
     runs-on: ubuntu-latest
     container:
-      image: robojackets/robocup-software:foxy
+      image: robojackets/robocup-software:humble
```

Comment:
did we push a humble tag of this image?

Reply:
i see that we did. it says on docker hub that I did actually, LOL. how did you guys manage that?

Reply:
Yup I made a humble tag on Sundays meeting

Reply:
You were still signed in on the shop computer so I logged in on your account